src/main.c

```diff
@@ -877,7 +888,7 @@ static gboolean info_start(int argc, char **argv)
 		goto out;
 	g_debug("input bundle: %s", bundlelocation);
 
-	res = check_bundle(bundlelocation, &bundle, !info_noverify, &error);
+	res = check_bundle(bundlelocation, &bundle, !noverify, &error);
```

Comment:
Instead of passing some verification options via r_context and noverify via an argument, I'd prefer replacing the verify argument by a bit-field enum (like `TestUpdateHandlerParams` in `test/update_handler.c`).
```
V_NO_VERIFY = BIT(0),
V_NO_CHECK_TIME = BIT(1),
```
The BIT macro should go in `utils.h`